.gitignore

```
@@ -10,3 +10,4 @@ Packages
 *.exe.settings
 *.exe.old
 *.dll
+artifacts
```

```suggestion
/artifacts/
```

This is to ensure only the `artifacts` folder at the root of the repo is ignored (and not `artifacts` files or folders which are not in the repo root. We have used this convention in other repos.

---

Sounds good

---

Forgot to mention, but can you remove `bin` and `obj` entries from the file as well? This is to make sure everyone cleans up the unneeded output.